en/developer-docs/docs/administer/configure-developer-portal-envs.md

```diff
@@ -0,0 +1,16 @@
+# Configure Exposed Environments to Developer Portal
```

Comment:
https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting 
Shall we change the file name accordingly.